engines/scumm/debugger.cpp

```diff
@@ -93,7 +94,15 @@ ScummDebugger::ScummDebugger(ScummEngine *s)
 	registerCmd("show",      WRAP_METHOD(ScummDebugger, Cmd_Show));
 	registerCmd("hide",      WRAP_METHOD(ScummDebugger, Cmd_Hide));
 
-	registerCmd("imuse",     WRAP_METHOD(ScummDebugger, Cmd_IMuse));
+#if defined(ENABLE_SCUMM_7_8)
```

Comment:
this looks like duplication for "imuse"

Reply:
Should I just keep "imuse" as the command name for tidiness?

Reply:
might be good idea, since they are exclude each other

Reply:
Yep, good point thanks, will do that

Reply:
it was rather unnecessary one call, this could looks like this:
if (_vm->_game.version < 7)
registerCmd("imuse", WRAP_METHOD(ScummDebugger, Cmd_IMuse));
#if defined(ENABLE_SCUMM_7_8)
else
registerCmd("imuse", WRAP_METHOD(ScummDebugger, Cmd_DiMuse));
#endif

Reply:
Very good, done